pkg/cmd/pr/merge/merge.go

```diff
@@ -131,6 +131,16 @@ func mergeRun(opts *MergeOptions) error {
 		return err
 	}
 
+	if opts.SelectorArg == "" {
```

Comment:
When `selectorArg` is empty we use the current local git branch to retrieve the PR so there is no need to recheck if the PR branch name matches the local git branch name.